scripts/gen-edpm-kustomize.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+
+# expect that the common.sh is in the same dir as the calling script
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. ${SCRIPTPATH}/common.sh --source-only
+
+if [ -z "$NAMESPACE" ]; then
+    echo "Please set NAMESPACE"; exit 1
+fi
+
+if [ -z "$KIND" ]; then
+    echo "Please set SERVICE"; exit 1
+fi
+
+if [ -z "$DEPLOY_DIR" ]; then
+    echo "Please set DEPLOY_DIR"; exit 1
+fi
+
+NAME=${KIND,,}
+
+if [ ! -d ${DEPLOY_DIR} ]; then
+    mkdir -p ${DEPLOY_DIR}
+fi
+
+pushd ${DEPLOY_DIR}
+
+cat <<EOF >kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+namespace: ${NAMESPACE}
+patches:
+- target:
+    kind: ${KIND}
+  patch: |-
+    - op: replace
+      path: /spec/deployStrategy/deploy
+      value: true
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/ansibleHost
+      value: ${EDPM_COMPUTE_NODE0_IP}
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/openStackAnsibleEERunnerImage
+      value: ${OPENSTACK_RUNNER_IMG}
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/node/ansibleSSHPrivateKeySecret
+      value: ${EDPM_ANSIBLE_SECRET}
+    - op: replace
+      path: /spec/nodes/edpm-compute-0/node/ansibleVars
+      value: |
+        tenant_ip: ${EDPM_COMPUTE_NODE0_IP}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/ansibleHost
+      value: ${EDPM_COMPUTE_NODE1_IP}
```

Comment:
Earlier, The EDPM setup is defaulting to single compute node. Now it is 2. Can we switch to 1?
Since crc is already resource hungry and I am not sure all the dev environment have enough resources.

Reply:
Ah, I can see we have https://github.com/raukadah/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenode_edpm_compute_0_full.yaml here also, we can export this one and it will work.

Reply:
I'm ok to default it to one, I will try removing the other node with kustomize